clientApp/src/app/components/supervisor/supervisor.component.sass

```diff
@@ -0,0 +1,13 @@
+.example-headers-align .mat-expansion-panel-header-title,
+.example-headers-align .mat-expansion-panel-header-description
```

Comment:
As we're using sass, you can write something like that:
```sass
.example-headers-align
    .mat-expansion-panel-header-title
        flex-basis: 0
    .mat-expansion-panel-header-description
        flex-basis: 0
        justify-content: space-between
        align-items: center
```

Reply:
Refactored